App/Product/frmProductModal.cs

```diff
@@ -32,8 +33,10 @@ private void btnSave_Click(object sender, EventArgs e)
             product.Category = new Category() { Id = Convert.ToInt32(cmbCategory.SelectedValue) };
             product.MetricUnit = new MetricUnit() { Id = Convert.ToInt32(cmbMetricUnit.SelectedValue) };
             product.MetricValue = this.txtMetricValue.Text;
+            product.Status = getStatus.Active;
 
             productController = new ProductRepository();
+            if (MessageBox.Show("Do you want to save the selected product?", "Save Product", MessageBoxButtons.YesNo, MessageBoxIcon.Question)==DialogResult.Yes)
```

Comment:
This code block is missing an opening and ending curly brace. This will result in executing the rest of the code regardless of the actions (i.e. YES or NO).